tests/common.go

```diff
@@ -7144,6 +7144,34 @@ func CreateAzureBucket(bucketName string, objectlock bool, mode Mode, retentionD
 
 }
 
+// UpdateConfigmap creates or updates an existing configmap with the given data
+func UpdateConfigmap(configMapName string, data map[string]string) error {
+	configmap, err := k8sCore.GetConfigMap(configMapName, "default")
+	if err != nil {
+		log.InfoD("Creating Configmap: %v", configMapName)
```

Comment:
You can add another another condition as follows:
```
if k8serrors.IsNotFound(err) {
  // Create the config map and update it
} else {
  // return error here
}
```

Reply:
Nice catch let me update it.